Dockerfile.amd64

```diff
@@ -1,12 +1,11 @@
-FROM frolvlad/alpine-glibc
+FROM alpine
 
 MAINTAINER Tom Denham <tom@tigera.io>
```

Comment:
nit: `MAINTAINER` has been deprecated, we should probably switch to `LABEL`

Reply:
Already done in another PR :smile: